nodes/config/ui_base.html

```diff
@@ -657,30 +825,90 @@
 
         divTabs.append('<div class="nrdb2-layout-helptext">' + c_('label.layoutMessage') + '</div>')
 
+        /** @type {DashboardItemLookup} */
         const pages = {}
+        /** @type {DashboardItemLookup} */
         const groupsByPage = {}
+        /** @type {DashboardItemLookup} */
         const widgetsByGroup = {}
-
+        const widgetsBySubflowDef = {}
+        const subflowDefsWithWidgets = {}
+        const widgetsBelongingToSubflow = new Map()
+        const subflowInstances = {}
```

Comment:
this is declared, and input into the `addGroupOrderingList` function, where it's never actually used? Not sure why linter isn't flagging it?

Reply:
As you might imagine, this was all very much a voyage of discovery. Once working, I meant to go back an remove try 1 (and try 2 and try n ;) )

Tidied up / removed a lot of this remnants in next comit

Reply:
resolved in [fef3ec1](https://github.com/FlowFuse/node-red-dashboard/pull/634/commits/fef3ec164c8042ce9e15e24550d7eb4faf84f0fb)